src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -165,6 +193,45 @@ private String getConfigValue(String configName) {
     return configEntryValueField.encode();
   }
 
+  private Future<String> initDatabase(Map<String, String> okapiHeaders, Vertx vertx) {
+    Promise<String> promise = Promise.promise();
+    String tenantId = okapiHeaders.get(OKAPI_TENANT);
+    vertx.executeBlocking(blockingFeature -> {
+      logger.info("KEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEK");
```

Comment:
What is it?

Reply:
> What is it?
My custom mark for logging the especially interested code part for me. Just forgot to remove. Already removed.